ArcGISRuntimeSDKQt_CppSamples/Layers/CreateAndSaveKmlFile/CreateAndSaveKmlFile.h

```diff
@@ -68,16 +69,22 @@ class CreateAndSaveKmlFile : public QObject
   Esri::ArcGISRuntime::Point m_point;
   Esri::ArcGISRuntime::Polyline m_polyline;
   Esri::ArcGISRuntime::Polygon m_polygon;
+  Esri::ArcGISRuntime::KmlStyle* m_pointStyle = nullptr;
+  Esri::ArcGISRuntime::KmlStyle* m_lineStyle = nullptr;
+  Esri::ArcGISRuntime::KmlStyle* m_polygonStyle = nullptr;
```

Comment:
Consider renaming these for clarity. The object model isn't intuitive, so I would be overly descriptive on the names to help users.

```c++
  Esri::ArcGISRuntime::KmlStyle* m_kmlStyleWithPointStyle = nullptr;
  Esri::ArcGISRuntime::KmlStyle* m_kmlStyleWithLineStyle = nullptr;
  Esri::ArcGISRuntime::KmlStyle* m_kmlStyleWithPolygonStyle = nullptr;
```